index.html

```diff
@@ -9,9 +10,28 @@
 	<link rel="stylesheet" href="css/style.css">
 	<!-- <link rel="stylesheet" href="css/style.min.css"> -->
 </head>
+
 <body class="body">
-	
+
+	<div class="container">
+		<div class="client-menu">
+			<a href="#" class="logo-blog-page">
+			    <img src="img/blog-page-logo.svg" alt="logo-blog-page" class="img-blog-page" width="32" height="32">
```

Comment:
Shu terda `alt` atributini to'g'ri tavsiflovchi so'zlar bilan yozish kerak. Skrin readerlar buni juda xunuk o'qiydi va bu ko'zi ojizlar uchun noqulay